LFGBulletinBoard/RequestList.lua

```diff
@@ -641,6 +671,10 @@ function GBB.PhraseMessage(msg,name,guid,channel)
 		dungeonList={}
 	end
 	
+	if GBB.DB.TagsZhtw and wordcount<2 then
+		wordcount=2
+	end
+
```

Comment:
Don't really get this either

Reply:
As mentioned example above.
Only one wordcount in above example. 
I have no idea why checking wordcount must larger than one. It's really a workaround.
May be we can have a better logic here.

Reply:
Yea agree the logic here is kind of janky. This work around makes more sense with your explanation of how Chinese works.